tests/queries/tests.py

```diff
@@ -2222,6 +2222,32 @@ def test_field_error_values_list(self):
             Tag.objects.values_list('name__foo')
 
 
+class IteratorQuerySetTests(TestCase):
```

Comment:
I'm trying to avoid adding any new classes to this file (there's a ticket to split it up, unsure if it's feasible...). I suggest `queries/test_iterator.py`.